### PR TITLE
Pull request - Milestone 4 Tic Tac Toe

### PR DESCRIPTION
**Pull request by bruna-genz**
In this branch, I created a Read Me file with the game instructions. Also, I created one file for each class of the game logic, for better organization.

### CODE REVIEW

**Comment by bruna-genz** [comment]
> Hello coders,
> 
> Great job done in this branch 
> However, your **README** is missing something.
> Please add **explicitly how to start and play the game.** also don't forget to **add the rules**
> 
> Kindly make those changes a submit a new code review request.
> Thanks.

**Comment by bruna-genz** [comment]
README file updated.

**Comment by samgaco** [comment]
Hi @bruna-genz :high_brightness: 

* You have updated the readme file on the development branch, it should have been updated in this pull request for which you are requiring the code review.

* Please address that and on your way, improve the styling by getting some inspiration from [here](https://github.com/othneildrew/Best-README-Template) or if you are feeling motivated you could also check this [course](https://www.udacity.com/course/writing-readmes--ud777). (for example, for the instructions on how to initiate the game you could use code styling for the lines of code attached. )

Please :pray:  submit back for another code review once this two changes are covered.

--------------

* ###### STATUS OF THE PROJECT: REQUESTED CHANGES :stop_sign:

File: README.md [hunk 1]
```diff
@@ -1,5 +1,32 @@
-# Microverse Project - Tic Tac Toe
+# Tic Tac Toe
+
+## Instalations
```
Comment (Ispirett) [inline README.md]: You can correct the spelling of this word.

>  Installation

**Pull request merged**